models/utils.py

```diff
@@ -0,0 +1,43 @@
+import operator
+from datetime import datetime
+from dmutils.formats import DATE_FORMAT, DATETIME_FORMAT
+
+
+format_datetime_string_as_date = lambda x: datetime.strptime(x, DATETIME_FORMAT).strftime(DATE_FORMAT)
```

Comment:
I'd put the functions we want to use in the CONFIG dictionaries in a separate file to make it easier to see what's available